src/scss/objects/_form-controls.scss

```diff
@@ -31,10 +31,10 @@
  $formControl-bgColor           : $white !default;
  $formControl-bgColor-disabled  : $color-disabled !default;
  $formControl-borderColor       : $color-border !default;
- $formControl-color             : $blue !default;
+ $formControl-color             : $orange !default;
+ $formControl-color--hover       : $orange--dark !default;
  $formControl-background-color  : $color-bg !default;
- $formControl-hover-color       : $blue--dark !default;
- $formControl-borderColor-hover : $blue !default;
+ $formControl-borderColor-hover : $orange !default;
```

Comment:
Should hover be a modifier like on L35? Appreciate it messes up the spacing

Reply:
Good shout – I'll update and check these